pkg/volume/iscsi/iscsi_util.go

```diff
@@ -243,7 +243,7 @@ func scanOneLun(hostNumber int, lunNumber int) error {
 	if written, err := fd.WriteString(scanCmd); err != nil {
 		return err
 	} else if 0 == written {
-		return fmt.Errorf("No data written to file: %s", filename)
+		return fmt.Errorf("No data written to file: %s ", filename)
```

Comment:
Please remove the extra space you added

Reply:
@jsafrane Thanks. I have removed the spaces added

PTAL, thanks